scripts/template_docsting_with_example.py

```diff
@@ -0,0 +1,77 @@
+# see more documentation guidelines online here: https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP)
+
+
+class SomeClass:
+    """A one line description of the Class.
+
+    A short paragraph providing more details.
+
+    Deprecation warning
+
+    Extended Summary
+
+    Parameters
+    ----------
+    scale_factor :class:`float`
+        The factor used for scaling.
+
+    Returns
+    -------
+    :class:`VMobject`
+        Returns self.
+
+    Tests
+    -----
```

Comment:
@behackl : Tests is now here, is the place ok?
And further, all Categories have currently one blank line in between.
Is that a good procedure?

Reply:
Looks good, in general.

I'm not sure I would actually add all sections to the docstring here, it might create the wrong impression that you have to include all of them when writing documentation. (Just an observation, not a blocking comment.)

Reply:
I have the same feeling!
Maybe there can be a note somewhere that they are optional.
Or even better:
We have two templates:
One here, that is minimal, and one extended with all possibilities that lives in the wiki